src/psygnal/_group.py

```diff
@@ -179,8 +131,8 @@ def connect_direct(
         """
 
         def _inner(slot: Callable) -> Callable:
-            for sig in self.signals.values():
-                sig.connect(
+            for sig in self._signals:
```

Comment:
Why is here without catch warnings?

Reply:
there are no warnings to catch here.  if you let me know what warnings you were expecting might happen, I'm happy to help clarify

Reply:
I see a similar pattern like in my first comment, but here without catching warnings.

Reply:
the warning we are specifically connecting in the `__init__` is a failed internal connection of `self._slot_relay`.  That's the only one we want to silence.  Any user-provided callables that are converted into strong references do need to be warned

Reply:
just to double check, is it clear to you here that the warning blocked in the context above, and the warnings (i thought) we are discussing here are *not* deprecation warnings?  the warning blocked above is [this one in `_weak_callback`](https://github.com/pyapp-kit/psygnal/blob/94d0a672ad963d3fa5c36d38ce3cc79029f04828/src/psygnal/_weak_callback.py#L232-L235)... 

just wanted to make sure you're not confusing the deprecation warnings that have been added elsewhere in this PR with the ones we're blocking and discussing here